fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/metadata/IndexTypes.java

```diff
@@ -147,6 +147,16 @@
      */
     public static final String BITMAP_VALUE = "bitmap_value";
 
+    /**
+     * An index remmebering minimum values in a way that can enumerate minima ordered by value and not (part of the) group.
+     */
+    public static final String PERMUTED_MIN = "permuted_" + FunctionNames.MIN;
+
+    /**
+     * An index remmebering maximum values in a way that can enumerate maxima ordered by value and not (part of the) group.
+     */
+    public static final String PERMUTED_MAX = "permuted_" + FunctionNames.MAX;
```

Comment:
Since there is some duplication, it would also be possible to have a `MIN_AND_MAX` type.

Reply:
Yes, you would.